lib/prometheus/client/data_stores/direct_file_store.rb

```diff
@@ -182,13 +185,15 @@ def process_id
 
           def aggregate_values(values)
```

Comment:
Could we also add a comment at the top of the method explaining that each entry in the `values` array is a tuple of "value, timestamp"? That makes the rest of the code a lot easier to follow